### DISCUSSION TITLE
citing the dataset

### DISCUSSION THREAD

**Discussion by glasseyes** [SuperDataScience-Community-Missions/MM0001-Operation-Mind-Shield#10] [Q&A] [answered]
The bottom of README.md says what to use to cite the dataset in the notebook.
But what do you do with that? It doesn't look right in either a code or text cell.

**Comment by Shaheerairaj** [comment, marked as answer]
Hi Dan. The original block of text was written in the BibTeX format. I've changed it to a markdown format.
